pom.xml

```diff
@@ -2671,6 +2671,15 @@
       </properties>
     </profile>
 
+    <profile>
+      <id>hadoop-3</id>
+      <properties>
+        <hadoop.version>3.1.0-SNAPSHOT</hadoop.version>
```

Comment:
Hey @steveloughran what is the possible release date for Hadoop 3.1.0?

Reply:
RC0 is up for testing right now! @leftnoteasy is managing the release